Dockerfile

```diff
@@ -6,15 +6,16 @@ RUN apt-get update
 RUN apt-get install -y curl git wget unzip libgconf-2-4 gdb libglu1-mesa fonts-droid-fallback python3
 RUN apt-get clean
 
-# Clone the flutter repo
-RUN git clone https://github.com/flutter/flutter.git /usr/local/flutter
+# Install flutter
+RUN curl -L https://storage.googleapis.com/flutter_infra_release/releases/stable/linux/flutter_linux_2.10.0-stable.tar.xz | tar -C /opt -xJ
 
 # Set flutter path
 # RUN /usr/local/flutter/bin/flutter doctor -v
-ENV PATH="/usr/local/flutter/bin:/usr/local/flutter/bin/cache/dart-sdk/bin:${PATH}"
+ENV PATH="/opt/flutter/bin:/opt/flutter/bin/cache/dart-sdk/bin:${PATH}"
 
 # Run flutter doctor
 RUN flutter doctor -v
+
 # Enable flutter web
 RUN flutter channel master
 RUN flutter upgrade
```

Comment:
just noticed this... that's gonna update your sdk to the latest :) (ok it's 2.10 now but in a near future..^^)